src/screens/qr/components/Toolbar.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {useCallback} from 'react';
+import {StyleSheet} from 'react-native';
+import {Box, Button} from 'components';
+import {useNavigation} from '@react-navigation/native';
+import {useI18n} from 'locale';
+import {BackButton} from '../views/BackButton';
+
+export interface ToolbarProps {
+  onIconClicked(): void;
```

Comment:
We also use this prop name on the datasharing toolbar but I think it is vague. Better would be `onClose`